packages/react-core/src/components/TextInput/examples/AltTextInput.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { TextInput } from '@patternfly/react-core';
+
+class altTextInput extends React.Component {
```

Comment:
The Class name should start with a capital letter.

Reply:
thanks! I'll fix this